openslides_backend/action/actions/meeting/mixins.py

```diff
@@ -0,0 +1,27 @@
+from typing import Any, Dict
+
+from ....permissions.management_levels import OrganisationManagementLevel
+from ....permissions.permission_helper import has_organisation_management_level
+from ....shared.exceptions import PermissionDenied
+from ....shared.patterns import Collection, FullQualifiedId
+from ...action import Action
+
+
+class MeetingPermissionMixin(Action):
+    def check_permissions(self, instance: Dict[str, Any]) -> None:
+        if has_organisation_management_level(
```

Comment:
Since you need this concept (checking for superadmin, then checking the CML) in multiple places, please create a new method for it, analogous to the normal permissions and the OML. `has_committee_management_level` should check if the user is superadmin and then if he has the expected CML.

Reply:
Add a has_committee_management_level.